src/DI/WebpackExtension.php

```diff
@@ -73,9 +79,7 @@ public function loadConfiguration()
 				new Statement(Client::class)
 			]);
 
-		$assetResolver = $builder->addDefinition($this->prefix('assetResolver'))
-			->setClass(AssetResolver\AssetResolverInterface::class)
-			->setFactory($config['assetResolver']);
+		$assetResolver = $this->setupAssetResolver($config);
 
 		if ($this->debugMode && $config['debugger']) {
 			$assetResolver->setAutowired(FALSE);
```

Comment:
on line 86 you are overriding property $assetResolver, it is counterintuitive because of combination with fluent interface